template.tpl

```diff
@@ -34,34 +26,35 @@ ___TEMPLATE_PARAMETERS___
 
 [
```

Comment:
After re-sorting the keys to compare, there aren't really any actual changes to the JSON introduced here.

Reply:
Yep - it should just be the one line change plus the format change in the export!